lib/monitor.js

```diff
@@ -126,20 +123,12 @@ module.exports = internals.Monitor = class {
                 streamObjs.unshift(new Utils.NoOp());
             }
 
-            const combinedStreamErrHandler = (err) => {
-                if (err) {
-                    console.error(`There was a problem (${err}) in ${reporterName} and it has been destroyed.`);
-                    console.error(err);
-                }
-            }
-
-<<<<<<< HEAD
-            const combinedStream = pipeline
```

Comment:
This logic was added in #2, but `pipeline()` returns the last stream, so it didn't allow `this.push()` to write to the first stream, which broke some tests. I'm proceeding assuming that `Pumpify.obj()` was updated to support node 10, but it's unclear from #2 what was being fixed so I'm not sure what other tests to do. cc @mistic

Reply:
Maybe this is what broke https://github.com/elastic/kibana/issues/43513?

Reply:
Have you tested this with kibana master?

Reply:
I just tested this with master and it works great, not sure if that means we can close https://github.com/elastic/kibana/issues/43513 once this is closed but I'll figure out for sure once this is ready to be included in Kibana, but that's going to be part of the Hapi upgrade.

Reply:
I just tested this with master and it works great, not sure if that means we can close https://github.com/elastic/kibana/issues/43513 once this is closed but I'll figure out for sure once this is ready to be included in Kibana, but that's going to be part of the Hapi upgrade.

Reply:
Thanks!